oci.go

```diff
@@ -129,16 +129,66 @@ func stopContainer(podStatus vc.PodStatus) error {
 }
 
 // processCgroupsPath process the cgroups path as expected from the
-// OCI runtime specification. It returns a complete path to the path
-// that should be created and used.
-func processCgroupsPath(ociSpec specs.Spec) (string, error) {
+// OCI runtime specification. It returns a list of complete paths
+// that should be created and used for every specified resource.
+func processCgroupsPath(ociSpec specs.Spec) ([]string, error) {
+	var cgroupsPathList []string
+
 	if ociSpec.Linux.CgroupsPath == "" {
-		return "", nil
+		return []string{}, nil
+	}
+
+	if ociSpec.Linux.Resources == nil {
+		return []string{}, nil
+	}
+
+	if ociSpec.Linux.Resources.Memory != nil {
```

Comment:
Well I cannot do that because for instance in case of memory, there is nothing to help making the relation between the string "memory" and the structure Memory. Am I missing something ?